iOS/Flows/Home/_New/new_ConversationViewController.swift

```diff
@@ -116,6 +118,17 @@ class new_ConversationViewController: FullScreenViewController,
         }
     }
 
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        guard let messageItem = self.dataSource.itemIdentifier(for: indexPath) else { return }
+
+        switch messageItem {
+        case .message(let messageID):
+            self.onSelectedThread?(self.conversation.cid, messageID)
```

Comment:
Letting the handler know that a thread was selected. Currently this let's you select any message, even if there's no replies on it currently.